server.py

```diff
@@ -25,6 +25,13 @@
 
 import csv
 
+def plain_textify(string):
+    """Convert a string to a UTF-8 encoded body with the text/plain
+    mimetype."""
+    resp = make_response(string)
+    resp.headers['Content-Type'] = 'text/plain; charset=utf-8'
```

Comment:
The [Response object](https://flask.palletsprojects.com/en/1.1.x/api/#flask.Response) may have been the slightly simple way to do this. i.e.,

```resp = Response(string, headers={'Content-Type': 'text/plain; charset=utf-8'})```

However, I think this is good enough too.